portfolio/src/main/webapp/script.js

```diff
@@ -17,12 +17,59 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['I\'m ready, I\'m ready, I\'m ready...', 'You know what\'s funnier than 24? 25!', 'East?! I thought you said Weest!', 'Squidwaard, I used your clarinet to unclog my toilet!'];
+      ['I\'m ready, I\'m ready, I\'m ready...', 
+      'You know what\'s funnier than 24? 25!', 
+      'East?! I thought you said Weast!', 
+      'Squidward, I used your clarinet to unclog my toilet!'];
 
-  // Pick a random greeting.
+  // Picks a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
-  // Add it to the page.
+  // Adds it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
-}
\ No newline at end of file
+}
+
+/**
+ * Reveals hidden description upon mouseover and hides the project title
+   text.
+ * @param {object} hoveredItem An anchor tag containing the
+      project image and text.
+ */
+function revealOnMouseover(hoveredItem) {
+  let toReveal = hoveredItem.getElementsByClassName('hidden')[0];
+  let toHide = hoveredItem.getElementsByClassName('visible')[0];
+
+  let background = hoveredItem.getElementsByClassName('background');
+
+  toReveal.style.visibility = 'visible';
```

Comment:
Given 'visible' and 'hidden' are used multiple places I would make them consts. 

"Declare all local variables with either const or let. Use const by default, unless a variable needs to be reassigned. The var keyword must not be used."

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Reply:
Since I'm using both of them only once in each method but in multiple methods, should I make 'hidden' and 'visible' global variables instead? Also, since both are single-worded strings, I feel it might be redundant to create variables for them.

Reply:
I found this in the style guide: 9.4.2.3 Use @const for constants variables
For global declarations where the const keyword would be used, if it were available, annotate the var declaration with @const instead (this is optional for local variables).

Although they are single-worded strings, if for some reason you need to change the string it helps to have it in 1 variable so you do not have to make the same change everywhere. Also you can imagine having a typo in one of the variables introducing bugs. Given these reasons, having 1 variable holding the string helps.

Reply:
I updated the two strings to be constants variables defined in the global scope.

Done!